doc/ref_model/chapters/chapter03.md

```diff
@@ -128,7 +129,20 @@ _**Example**: a virtual compute descriptor as defined in TOSCA Simple Profile fo
 | `subnet`       | network address of the subnet                                            |
 | `acceleration` | key/value pairs for selection of the appropriate acceleration technology |
 
-<p align="center"><b>Table 3-4:</b> Attributes of network resources.</p>
+<p align="center"><b>Table 3-4:</b> Attributes of network resources</p>
+
+<a name="3.2.5"></a>
+### 3.2.5 Availability Zone
+An Availability Zone is a logical pool of physical resources (e.g. compute, block storage, network).  These logical pools segment the physical resources of a cloud based on factors chosen by the cloud operator. The cloud operator may create availability zones based on location (rack, datacenter), or indirect failure domain dependencies like power sources.  Workloads can leverage availability zones to avoid sharing failure domains for a workload, and thus increase its fault-tolerance.
```

Comment:
"Workloads can leverage availability zones to avoid sharing failure domains for a workload, and thus increase its fault-tolerance."
In the previous sentence AZs are defined as either "location" or "failure domains" and hence I suggest that we chnage to, say,:
"Workloads can leverage availability zones to utilise multiple locations or avoid sharing failure domains for a workload, and thus increase its fault-tolerance."

Reply:
@pgoyal01 @kedmison, Utilize multiple location , introduce the concept of HA or DR which on of the target of AZ 
So I will go with @pgoyal01 modification

Reply:
good point. applied.